_inc/lib/debugger/class-jetpack-debug-data.php

```diff
@@ -349,4 +339,25 @@ public static function debug_data() {
 
 		return $debug_info;
 	}
+
+	/**
+	 * Returns a human readable string for which user is the master user.
+	 *
+	 * @return string
+	 */
+	private function human_readable_master_user() {
+		$master_user = Jetpack_Options::get_option( 'master_user' );
+
+		if ( ! $master_user ) {
+			return 'No master user set.';
+		}
+
+		$user = new WP_User( $master_user );
+
+		if ( ! $user ) {
+			return 'Master user no longer exists. Please disconnect and reconnect Jetpack.';
+		}
+
+		return sprintf( '#%1$d %2$s (%3$s)', $user->ID, $user->user_login, $user->user_email ); // Format: "#1 username (user@example.com)".
```

Comment:
Can we make this part into its own method and also used it for when we are display the current user?

Reply:
24eb226ce